config/githubauth/packit.yml

```diff
@@ -0,0 +1,101 @@
+## The name of the docker network that containers will be attached to.
+## If you want to proxy Packit to the host, you will need to
+## arrange a proxy on this network, or use dev_mode in the web section
+## below.
+## Prefix for container names; we'll use {container_prefix}-(container_name)
+container_prefix: packit
+
+## Set this flag to true to prevent use of --volumes in the cli to remove
+## volumes on stop
+protect_data: false
+
+## Docker org for images
+repo: mrcide
+
+## The name of the docker network that containers will be attached to.
+## If you want to proxy Packit to the host, you will need to
+## arrange a proxy on this network
+network: packit-network
+
+## Names of the docker volumes to use:
+##
+## outpack: stores the outpack metadata
+## proxy_logs: stores logs from the reverse proxy (only used if proxy is given)
+## (More volumes are anticipated as the tool develops)
+volumes:
+  outpack: outpack_volume
+  proxy_logs: packit_proxy_logs
+
+outpack:
+  ## Initialise an outpack directory by cloning from github
+  initial:
+    ##  If using a private repo, then use an ssh url and provide ssh keys in the
+    ## "ssh" section above.
+    url: https://github.com/reside-ic/orderly3-example.git
+  server:
+    name: outpack_server
+    tag: main
+  migrate:
+    name: outpack.orderly
+    tag: main
+
+packit:
+  api:
+    name: packit-api
+    tag: mrc-4922
+  app:
+    name: packit
+    tag: mrc-4922
+  db:
+    name: packit-db
+    tag: main
+    user: packituser
+    password: changeme
+  auth:
+    enabled: true
+    enable_github_login: true
+    expiry_days: 1
+    github_api_org: mrc-ide
+    github_api_team: packit
+    # Details of your Github OAuth app, which should be kept in the vault. The app's Authorization callback url must
+    # have the same root as the packit_api_root specified below, and should be of the form
+    # {PACKIT_API_ROOT}/login/oauth2/code/github
+    github_client:
+      id: VAULT:secret/packit/githubauth/auth/githubclient:id
+      secret: VAULT:secret/packit/githubauth/auth/githubclient:secret
+    jwt:
+      # Secret used to generate JWT tokens
+      secret: VAULT:secret/packit/githubauth/auth/jwt:secret
+    oauth2:
+      redirect:
+        # Root url which OAuth2 app will be requested to use to redirect back to packit api - must match root of OAuth2 app's registered url
+        packit_api_root: "https://localhost/packit/api"
```

Comment:
I imagine this will be affected by any changes we make to support "multiple sites hosted on one server", but if these urls always share a common stem and if the remaining bits are packit details can we just save the stem here (so `https://localhost/` here with `packit-deploy` adding the `packit/api` and `redirect` bits)

Reply:
As discussed, these values may have a different pattern if running a proxy with a different nginx config, so leaving this as is for the moment.